etc/init.d/contextBroker.ubuntu

```diff
@@ -0,0 +1,184 @@
+#! /bin/sh
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
```

Comment:
Digging into this, it seems that the official ocumentation about the BEGING/END INIT INFO system doesn't say anything about this (see http://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/initscrcomconv.html)

However, looking to some random examples, it seems to be ok having the license header before this block (as it is now in your code).

https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1/src/packages/deb/init.d/hadoop-jobtracker
www.virtualbox.org/svn/vbox/trunk/src/VBox/ValidationKit/utils/TestExecServ/linux/vboxtxs.sh

So, I'd suggest to have a look into the file in /etc/init.d directory in your Ubuntu system (I'm afraid I cannot do it myself, as I don't use Ubuntu :)) and, if they are using the same pattern, let as it it now.